particles/TestingExample/ComputePeopleStatsTest.kt

```diff
@@ -0,0 +1,44 @@
+package arcs.unit.test.example
+
+import arcs.sdk.testing.TestingCollection
+import arcs.sdk.testing.TestingSingleton
+import com.google.common.truth.Truth.assertThat
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runBlockingTest
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+
+/**
+ * Example of particle Unit Testing.
+ */
+@ExperimentalCoroutinesApi
+@RunWith(JUnit4::class)
+class ComputePeopleStatsTest {
+
+    @Test
+    fun testMedianReactsToInputChanges() = runBlockingTest {
+        val particle = ComputePeopleStats()
+
+        val peopleHandle = TestingCollection<ComputePeopleStats_People>(particle, "people")
+        val statsHandle = TestingSingleton<ComputePeopleStats_Stats>(particle, "stats")
```

Comment:
Is there a way to have these handles be created automatically?

Reply:
Obviously, coding is magic, isn't it? 🧙and now it's generated!